test/mitmproxy/data/har_files/chrome.json

```diff
@@ -194,10 +194,6 @@
                 [
                     "x-cache",
                     "Hit from cloudfront"
-                ],
-                [
-                    "content-length",
-                    "23866"
                 ]
```

Comment:
We no longer add a `content-length` header if it does not exist in the recorded response.